components/automate-chef-io/content/docs/restore.md

```diff
@@ -0,0 +1,119 @@
++++
+title = "Restore"
+description = "Procedures for Restoring Chef Automate Data"
+date = 2018-03-26T15:27:52-07:00
+draft = false
+bref = ""
+toc = true
+[menu]
+  [menu.docs]
+    parent = "get_started"
+    weight = 80
++++
+
+Restore Chef Automate from a [filesystem backup]({{< ref "restore.md#restore-from-a-filesystem-backup" >}}) or an [Amazon S3 bucket backup]({{< ref "restore.md#restore-from-an-s3-backup" >}}).
+
+## Prerequisites
+
+1. On the restore host, download and unzip the Chef Automate command-line tool:
+
+   ```shell
+        curl https://packages.chef.io/files/current/latest/chef-automate-cli/chef-automate_linux_amd64.zip | gunzip - > chef-automate && chmod +x chef-automate
+    ```
+
+1. **Filesystem backups** require access for Chef Automate to a backup directory in the [configured location]({{< ref "backup.md#backup-to-a-filesystem" >}}).
```

Comment:
Clarify that this is about restoring, the main point expressed here is about backing up.